packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js

```diff
@@ -6,6 +6,12 @@ import { IntlProvider } from 'react-intl';
 
 import Blocks from '../index';
 
+jest.mock('@strapi/helper-plugin', () => ({
+  useLibrary: () => ({ components: { 'media-library': jest.fn() } }),
+  prefixFileUrlWithBackendUrl: jest.fn(),
+  pxToRem: jest.fn(),
```

Comment:
How come you've had to mock these two? :)

Reply:
`pxToRem` is used inside Toolbar, selector style, so have to mock it, otherwise it throws error. 
`prefixFileUrlWithBackendUrl` is not needed removed it.

Reply:
Okay, but why? It's just `const pxToRem = (px: number) => `${px / 16}rem`` it shouldn't throw errors...?

Reply:
not sure, I get this error, TypeError: (0 , _helperplugin.pxToRem) is not a function.

We have jest.requireActual('@strapi/helper-plugin') in most of the tests, do we need to requireActual to not mock this then? I think mocking is lighter than importing helperplugin module.

Reply:
If you're not mocking anything in the `helper-plugin` you shouldn't need to require the actual 🤔  let me look at the test...

Reply:
Yeah, so your mock should look like this:

```ts
jest.mock('@strapi/helper-plugin', () => ({
  ...jest.requireActual('@strapi/helper-plugin'),
  useLibrary: () => ({ components: { 'media-library': jest.fn() } }),
}));
```

So you're just mocking a specific export of the module as opposed to the whole module :)

Reply:
right